src/transport/SessionManager.cpp

```diff
@@ -162,6 +162,8 @@ CHIP_ERROR SessionManager::PrepareMessage(const SessionHandle & sessionHandle, P
     FabricIndex fabricIndex;
 #endif // CHIP_PROGRESS_LOGGING
 
+    PeerAddress destination_address;
```

Comment:
nit. Recommend using camel case for the variable, e.g. destAddress, to be consistent.

Reply:
I was using existing naming, fwiw.  Trying to minimize the churn.